develop/plone/searching_and_indexing/external-catalogs.rst

```diff
@@ -0,0 +1,83 @@
+=======================
+Using external catalogs
+=======================
+
+.. admonition:: Description
+
+    The Plone catalog can be extend to use external catalogs like Solr or Elasticsearch.
+    Add-ons like collective.solr use that to hook into the catalog API and do some Indexing outside of Plone in Solr,
+    which increases performance and flexibility of indexing a lot.
+
+.. contents :: :local:
+
+
+Implement IIndexingQueueProcessor
+---------------------------------
+
+To hook into the catalog one can implement the IIndexingQueueProcessor
+`interface from Products.CMFCore <https://github.com/zopefoundation/Products.CMFCore/blob/master/Products/CMFCore/interfaces/_tools.py>`_.
+
+
+.. versionadded:: 5.1
+   For Plone < 5.1 you need to use the
+   `interfaces from collective.indexing <https://github.com/plone/collective.indexing/blob/master/src/collective/indexing/interfaces.py>`_ package!
+
```

Comment:
Is it maybe an idea, to write explicit something like:

For Plone versions below 5.1 you have to ... bla bla bla ?

I missed the < twice during reading, so maybe is would be better to be more clear in saying it out :)